drivemount/drive-list.c

```diff
@@ -200,19 +199,20 @@ drive_list_add (GtkContainer *container, GtkWidget *child)
     g_return_if_fail (DRIVE_IS_BUTTON (child));
 
     if (GTK_CONTAINER_CLASS (drive_list_parent_class)->add)
-	(* GTK_CONTAINER_CLASS (drive_list_parent_class)->add) (container,
-								child);
+        (* GTK_CONTAINER_CLASS (drive_list_parent_class)->add) (container,
+                                                                child);
```

Comment:
what is the benefit of breaking the line here?

Reply:
This wasn't my idea :)
Any way, i will fix it.